src/Contacts/Threads/TSContactThread.m

```diff
@@ -32,7 +32,12 @@ + (instancetype)getOrCreateThreadWithContactId:(NSString *)contactId
         [SignalRecipient recipientWithTextSecureIdentifier:contactId withTransaction:transaction];
 
     if (!recipient) {
-        recipient = [[SignalRecipient alloc] initWithTextSecureIdentifier:contactId relay:relay supportsVoice:YES];
+        recipient =
+            [[SignalRecipient alloc] initWithTextSecureIdentifier:contactId
+                                                            relay:relay
+                                                    supportsVoice:YES
+                                                   // Default to NO; ContactsUpdater will try to update this property.
```

Comment:
Agree. 

Some relevant reading that might shape our solutions https://github.com/YapStudios/YapDatabase/wiki#intermediate